keyboards/primekb/prime_o/prime_o.h

```diff
@@ -19,22 +19,22 @@
 #include "quantum.h"
 
 #define LAYOUT( \
-	K000, K001, K002, K003, K004, K005, K006, K007, K010, K011, K012, K013, K014, K015, K016, K017, \
-	K020, K021, K022, K023, K024, K025, K026, K027, K030, K031, K032, K033, K034, K035, K036, K037, \
-	K040, K041, K042, K043, K044, K045, K046, K047, K050, K051, K052, K053, K054, K055, K056, K057, \
-	K060, K061, K062, K063, K064, K065, K066, K067, K070, K071, K072, K073, K074, K075, K076, K077, \
-	K080, K081, K082, K083, K084, K085, K086, K087, K090, K091, K092, K093, K094, K095, K096, K097  \
+	K001, K008, K013, K018, K023, K029, K035, K041, K046, K052, K058, K064, K071, K076, K081, K086, \
```

Comment:
Why the change to the matrix here?

Reply:
I did it so that the numbers match up with the actual numbers written on the pcb.